src/main/java/com/hbm/blocks/machine/rbmk/RBMKDebrisBurning.java

```diff
@@ -36,12 +37,16 @@ public void updateTick(World world, int x, int y, int z, Random rand) {
 			}
 			
 			ForgeDirection dir = ForgeDirection.getOrientation(rand.nextInt(6));
+			Block block = world.getBlock(x + dir.offsetX, y + dir.offsetY, z + dir.offsetZ);
 
-			if(rand.nextInt(7) == 0 && world.getBlock(x + dir.offsetX, y + dir.offsetY, z + dir.offsetZ) == Blocks.air) {
+			if(rand.nextInt(10) == 0 && block == Blocks.air) {
 				world.setBlock(x + dir.offsetX, y + dir.offsetY, z + dir.offsetZ, ModBlocks.gas_meltdown);
 			}
 			
-			if(rand.nextInt(100) == 0) {
+			//Foam helps stop the fire; 1.66% chance every 100-120 seconds for one side
+			int chance = block == ModBlocks.foam_layer || block == ModBlocks.block_foam ? 10 : 100;
```

Comment:
Shouldn't this include boron sand too?

Reply:
At the time, it made sense to restrict it just to foam, but in retrospect it doesn't really make much sense besides foam being cheaper I suppose. If necessary I can always change it later today.